src/nycdb/dataset.py

```diff
@@ -64,9 +65,32 @@ def index(self):
         else:
             logging.debug('no index files exist for this dataset')
 
+    def _get_transformed_dataset_rows_using_module(self, schema, dataset_transformations_module):
+        rows_func = getattr(dataset_transformations_module, schema['table_name'], None)
+        if rows_func:
+            return rows_func(self)
+        rows_func = getattr(dataset_transformations_module, self.name, None)
+        if rows_func:
+            return rows_func(self, schema)
+        return None
+
+    def _get_transformed_dataset_rows(self, schema):
```

Comment:
I find this section confusing. and, this is not to say what we had before was good -- that was also confusing!

i think we should deal with transformations differently. Perhaps we could try something like having a set of common transformations and then link them together in dataset yaml file:
``` yaml
     transformations: 
         - csv
         - bbl 
``` 

regardless of that proposal, I think we should refactor the transformer/transformations code (a lot of the confusion was my fault when I introduced the option of using a transformer named for either the dataset or the schema name)

Reply:
Ah good idea, I did find this code confusing to write, so I think it'd be nice to be more explicit with the transformations if possible!

Reply:
Hmm, so it seems like we should probably hold off on merging this PR until this whole transformations API is changed.  Do you want to write up a PR for that or should I?